biscuit-auth/src/datalog/mod.rs

```diff
@@ -707,6 +706,12 @@ impl FactSet {
         self.inner.values().fold(0, |acc, set| acc + set.len())
     }
 
+    pub fn is_empty(&self) -> bool {
```

Comment:
why add this method? It is not used anywhere

Reply:
Because clippy is … pedantic. It says, whenever there is a public `len()` method there should be a public `is_empty()` method because it might be to compute [less costly](https://rust-lang.github.io/rust-clippy/master/#len_without_is_empty). But I can also add an ignore if you prefer.

Reply:
right, that method is not useful right now